extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -55,6 +56,12 @@ export class BaseSlides extends BaseCarousel {
     this.shouldAutoplay_ = this.hasAutoplay_ && this.isLoopingEligible();
 
     if (this.shouldAutoplay_) {
+      this.registerAction('toggleAutoPlay', invocation => {
+        const args = invocation.args;
+        if (args) {
```

Comment:
If we go with just `toggle` action (which is fine), we should making `toggleOn` arg optional. If not provided, it would do a toggle based on current state.

Reply:
Done